src/conf.c

```diff
@@ -830,6 +830,17 @@ int config__read_file_core(struct mosquitto__config *config, bool reload, const
 #else
 					log__printf(NULL, MOSQ_LOG_WARNING, "Warning: Bridge and/or TLS support not available.");
 #endif
+				}else if(!strcmp(token, "bridge_server_name")){
+#if defined(WITH_BRIDGE) && defined(WITH_TLS)
+					if(reload) continue; // FIXME
```

Comment:
This came as copy-n-paste -- most of the "if(reload) continue;" lines in the file have a "FIXME" comment.  I retained it as part of following the pattern.